### PR TITLE
Added a new superhero level

### PR DESCRIPTION
**Pull request by arjunpandey2003**
My steps to solve it:
Added necessary files

Deploy Link: http://rawgit.com/arjunpandey2003/labyrinth/master/superheroes.html

Efforts of @arjunpandey2003 and @samyakbvs

### CODE REVIEW

File: js/levelsSuperhero.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ +/* global NullTile */
+ +/* global door */
+ +/* global PlayerStartsAt */
+ +
+ +// defined in /js/level.js
+ +/* global Level */
+ +
+ +// createTestLevel is used in /js/gui.js ignoring the error
+ +// ignore JSHintBear
+ +function createTestLevel()
```
Comment (niccokunzmann) [inline js/levelsSuperhero.js]: Please add this code to to levels.js and rename the function to createSuperheroLevel()

File: js/tilesSuperhero.js [hunk 2]
```diff
@@ -0,0 +1,227 @@
+const NullTile = {
+ +  canEnterFromTheTop: function(player) {return false;},
+ +  canEnterFromTheBottom: function(player) {return false;},
+ +  canEnterFromTheRight: function(player) {return false;},
+ +  canEnterFromTheLeft: function(player) {return false;},
+ +  playerLeaves: function(player){},
+ +  placeAtIn: function(){return this;},
+ +  showIn: function() {},
+ +  visit: function() {},
+ +  viewFromTheTop: function() {},
```
Comment (niccokunzmann) [inline js/tilesSuperhero.js]: I will not allow this duplicated code. Please add your tile to the tiles.js file.

File: superheroes.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+ <div class="row">
+     <div class="col m4 s8">
+       <div class="card blue-grey darken-1">
+         <div class="card-content white-text">
+           <span class="card-title">Statistics</span>
+           <p>Player Moved : <span id="playerMoves">0</span> Times</p>
+           <p></i>Lv.3 Forest by Arjun and Samyak</i></p>
+         </div>
+         <div class="card-action">
+           <a href="http://github.com/yashkumarverma/">UI by Yash Kumar Verma</a>
```
Comment (niccokunzmann) [inline superheroes.html]: Please do not copy the index file #161 This needs to be reverted and it is almost impossible to revert this act of copying.

**Comment by niccokunzmann** [comment]
First of all: Thanks for taking the time to create such a level!

**Comment by duttaditya18** [comment]
Doesn't seem to be working.

Got the following errors : 
```
tilesSuperhero.js:2:1 : SyntaxError: expected property name, got '+'
levelsSuperhero.js:13:1 : SyntaxError: expected property name, got '+'
```

**Comment by samyakbvs** [comment]
We'll be making a new PR in some time, we aren't dropping this idea

**Comment by samyakbvs** [comment]
Yeah we'll be solving that first then we'll add the new level @abishekvashok

**Pull request closed without merging**